tests/composite-proofs/bbs-signature-and-accumulator.spec.ts

```diff
@@ -1,25 +1,28 @@
 import { initializeWasm } from '@docknetwork/crypto-wasm';
-import { stringToBytes } from '../utils';
+import { checkResult, stringToBytes } from '../utils';
 import {
   Accumulator,
   CompositeProofG1,
-  KeypairG2,
   MetaStatement,
   MetaStatements,
   PositiveAccumulator,
   ProofSpecG1,
-  Signature,
-  SignatureG1,
-  SignatureParamsG1,
   Statement,
   Statements,
   Witness,
   WitnessEqualityMetaStatement,
   Witnesses
 } from '../../src';
+import {
+  KeyPair,
+  Signature,
+  SignatureParams,
+  buildStatement,
+  buildWitness,
+} from '../scheme'
 import { InMemoryState } from '../../src/accumulator/in-memory-persistence';
 
-describe('Proving knowledge of 1 BBS+ signature and a certain message in the accumulator', () => {
+describe('Proving knowledge of 1 signature and a certain message in the accumulator', () => {
```

Comment:
A general comment on this and other tests using `scheme.ts`, have the name of the sig scheme in `describe` to make it easier to detect failing tests from logs